code-challenges.test.js

```diff
@@ -43,4 +187,58 @@ const cubeAndSum1 = [2, 3, 4]
 const cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
 
+describe("sumAndCubed",()=>{
+  it("return a cubed sum of numbers input as an array", ()=>{
+    expect(sumAndCubed(cubeAndSum1)).toEqual(99)
+    expect(sumAndCubed(cubeAndSum2)).toEqual(1125)
+  })
+})
+
+// Good Fail:
+// FAIL  ./code-challenges.test.js
+// sumAndCubed
+//   ✕ return a cubed sum of numbers input as an array (1 ms)
+
+// ● sumAndCubed › return a cubed sum of numbers input as an array
+
+//   ReferenceError: sumAndCubed is not defined
+
+//     106 | describe("sumAndCubed",()=>{
+//     107 |   it("return a cubed sum of numbers input as an array", ()=>{
+//   > 108 |     expect(sumAndCubed(cubeAndSum1)).toEqual(99)
+//         |     ^
+//     109 |     expect(sumAndCubed(cubeAndSum2)).toEqual(1125)
+//     110 |   })
+//     111 | })
+
+//     at Object.expect (code-challenges.test.js:108:5)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+// Snapshots:   0 total
+// Time:        0.727 s
+// Ran all test suites.
+
+
 // b) Create the function that makes the test pass.
+//Pseudo Code:
+//1. Create a function called sumAndCubed that takes in an array as input and returns a number as output
+//2. take each element of the given array and add them together, storing that value in a variable called sum
+//3. then return the value of sum cubed
+
+const sumAndCubed = (array) => {
+  return sum = (array[0] ** 3) + (array[1] ** 3) + (array[2] ** 3)
+}
+
```

Comment:
Refactor: What is a high order function that will return an array with each value cubed? How can you dynamically access the values within the array?